src/date-timer/_date-timer.scss

```diff
@@ -1,11 +1,11 @@
-.countdown {
+.date-timer {
   display: grid;
   grid-template-columns: repeat(auto-fit, 60px);
 
   width: 100%;
 }
 
-.countdown-box {
+.date-timer-box {
```

Comment:
Let's add the following property for `date-timer-box__count` class:

```
font-variant-numeric: tabular-nums;
```

It prevents the jump due to different digit widths